lib/state_change_syncer.go

```diff
@@ -750,23 +750,6 @@ func (stateChangeSyncer *StateChangeSyncer) SyncMempoolToStateSyncer(server *Ser
 			// If the transaction successfully connected, we update mempoolTxUtxoView to the copied view.
 			if err == nil {
 				mempoolTxUtxoView = copiedView
-			} else {
```

Comment:
If err is not nil, we should glog an error maybe?

Reply:
Agreed. However, I think we should only glog here, similarly to the block producer, which skips over failing transactions now. But I'm not fully familiar with the use of this function, so I may be wrong here.